charts/alfresco-search-service/charts/alfresco-insight-zeppelin/values.yaml

```diff
@@ -37,8 +39,13 @@ livenessProbe:
   initialDelaySeconds: 130
   periodSeconds: 20
   timeoutSeconds: 10
-# -- Apply your secret file in k8s environment to access quay.io images (Example: https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/SECRETS.md)
-# Global definition of Docker registry pull secret which can be overridden from parent ACS Helm chart(s)
+environment:
+  ZEPPELIN_MEM: "-Xms512m -Xmx1g"
+  ZEPPELIN_INTP_MEM: "-Xms512m -Xmx1g"
+# -- Apply your secret file in k8s environment to access quay.io images (see:
+# https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/SECRETS.md)
+# Global definition of Docker registry pull secret which can be overridden from
+# parent ACS Helm chart(s)
```

Comment:
move it on top of alfrescoRegistryPullSecrets so it doesn't get rendered unproperly